### PR TITLE
Feature invoice item repo

### PR DESCRIPTION
**Pull request by abreaux26**
**What did you change?**
Created Invoice Item and Invoice Item Repo

**Next Steps**
Create Transaction and Transaction Repo

**Known Issues**
N/A

### CODE REVIEW

**Comment by jbranham1** [comment]
@abreaux26 Let's look at this today and see if I can help!

File: lib/invoice_item_repository.rb [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    all.delete(find_by_id(id))
+  end
+
+  private
+
+  def max_invoice_item_id
+    all.max_by(&:id).id
+  end
+
+  def get_info(row)
```
Comment (jbranham1) [inline lib/invoice_item_repository.rb:71]: Just curious if this would resolve itself if there was like a convert_unit_price method?

**Comment by jbranham1** [comment]
https://stackoverflow.com/questions/30932732/what-is-meant-by-assignment-branch-condition-size-too-high-and-how-to-fix-it
https://redgreenrepeat.com/2017/01/20/understanding-assignment-branch-condition/
@abreaux26 Links that could potentially be helpful!

File: test/invoice_item_test.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
+require './test/test_helper'
+require './lib/invoice_item'
+require 'bigdecimal'
+
+class InvoiceItemTest < Minitest::Test
+  def setup
```
Comment (abreaux26) [inline test/invoice_item_test.rb:6]: @jbranham1 @aschwartz1 did we ignore this in item_test.rb?

File: test/invoice_item_test.rb [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        updated_at: Time.now
+      }, @repository
+    )
+  end
+
+  def test_it_exists
+    assert_instance_of InvoiceItem, @invoice_item
+  end
+
+  def test_readable_attributes
```
[the remark this answers is not in the document]
Comment (abreaux26) [reply test/invoice_item_test.rb:25]: I honestly have no idea if I can fix this..

**Review by jbranham1 — APPROVED** [review]
Looks great!

**Pull request merged**